### PR TITLE
Upgrade Django to 2.2.11 and 1.11.29

### LINKED ISSUE #103: Upgrade Django to 2.2.11 and 1.11.29 (3/4)
> Django versions 3.0.4, 2.2.11 and 1.11.29 will be released on Wednesday, March 4, 2020 around 1000 UTC.They will fix a security defect with severity “high”.

See: https://travis-ci.org/azavea/docker-django/builds/645553085 (all builds failed, but they look to be PostgreSQL client library specific)

### PR DESCRIPTION
**Pull request by hectcastro**
Upgrades Django to 2.2.11 and 1.11.29 to apply a high severity security patch. The fixes won't be available until 3/4 though. This is mostly to resolve the PostgreSQL specific issues before 3/4.

> Django versions 3.0.4, 2.2.11 and 1.11.29 will be released on Wednesday, March 4, 2020 around 1000 UTC.They will fix a security defect with severity “high”.

Resolves https://github.com/azavea/docker-django/issues/103

---

**Testing**

See: https://travis-ci.org/azavea/docker-django/builds/656410899

### CODE REVIEW

File: Dockerfile-alpine.template [hunk 1]
```diff
@@ -7,12 +7,12 @@ COPY requirements.txt /tmp/
 
 RUN set -ex \
     && apk add --no-cache --virtual .build-deps \
-        --repository http://dl-cdn.alpinelinux.org/alpine/v3.9/main \
+        --repository http://dl-cdn.alpinelinux.org/alpine/v3.11/main \
```
Comment (hectcastro) [inline Dockerfile-alpine.template]: Due to the moving pointer nature of Docker image tags, we are on Alpine 3.11.

File: LICENSE [hunk 2]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2017 Azavea Inc.
+   Copyright 2020 Azavea Inc.
```
Comment (coalkettler) [inline LICENSE:189]: Good catch, I'll keep an eye out for this in license files as well.

File: Dockerfile-alpine.template (hunk 1 above)
Comment (coalkettler) [reply Dockerfile-alpine.template]: I'm in favor of pinning to `alpine3.11` in the `FROM` line as well, similarly to how we pin to `buster` in the `slim` variant. If we do that we might be able to remove `--repository http://dl-cdn.alpinelinux.org/alpine/v3.11/main` options altogether. I'm seeing the same output from `apk add` when running it locally.

**Review by coalkettler — COMMENTED** [review]
Left a comment on a possible improvement. Back to you! ⚾️

File: Dockerfile-alpine.template (hunk 1 above)
Comment (hectcastro) [reply Dockerfile-alpine.template]: I agree. FYI, the Alpine version pinned package repositories were added by https://github.com/azavea/docker-django/commit/6a5baa8d1c2a609dccd61c43952811624b83fc37#diff-5c587da9015d91ff9da4528825b5add6.

Should be fixed by:

- 81f3a885381abe1915d6f5ca904a913d387ff0a0
- cbc4759d5fa6b33f8394549795bd59ba002552b9
- dbd07639b97b05509956bcacdec0477a3966dfed

**Review by coalkettler — APPROVED** [review]
Ah, the `--repository` lines make sense in context of https://github.com/azavea/docker-django/commit/6a5baa8d1c2a609dccd61c43952811624b83fc37#diff-5c587da9015d91ff9da4528825b5add6. It's possible we could run into something like that again but for now removing them makes sense.

This all looks great, merge away!

**Pull request merged**